PTBR/DynamicRadio_PTBR.txt

```diff
@@ -1,85 +1,85 @@
-    AEBS_fog_0 = "Light fog.",
-    AEBS_fog_1 = "Thick fog.",
-    AEBS_fog_2 = "Very thick fog.",
-    AEBS_weather_0_a = "WARNING: Period of weather predicted to start in the %s... ",
-    AEBS_weather_0_b = "Weather continues throughout the day... ",
-    AEBS_weather_0_c = "Weather period predicted to end in the %s... ",
-    AEBS_weather_predicted = "expecting ",
-    AEBS_weather_and_a = " and a ",
-    AEBS_weather_light_moderate = "expecting light to moderate rain...",
-    AEBS_weather_heavy_rain = "showers and or heavy rain",
-    AEBS_weather_storm = "thunderstorm",
-    AEBS_weather_tropical = "tropical storm",
-    AEBS_weather_blizzard = "blizzard",
-    AEBS_weather_snowfall = " with a chance on snowfall.",
-    AEBS_weather_storm_C = "Thunderstorm",
-    AEBS_weather_tropical_C = "Tropical storm",
-    AEBS_weather_blizzard_C = "Blizzard",
-    AEBS_weather_warning = "Severe Weather Warning. %s Imminent. ETA %s days...",
+    AEBS_Intro = "<bzzt> Cinco Zero Dois. <wzzt> Sistema de transmissão automatizado. <bzzt>",
+    AEBS_Choppah = "Atividade Aérea detectada.",
+    AEBS_Power_1 = "Knox Power Grid: Flutuações de energia detectadas.",
+    AEBS_Power_2 = "Knox Power Grid: Sistemas falhando. Rede comprometida.",
+    AEBS_Power_3 = "Knox Power Grid: Apagão.",
+    AEBS_Pre_today = "Hoje,",
+    AEBS_Pre_tomorrow = "Amanhã,",
+    AEBS_Pre_dayafter = "Depois de amanhã,",
+    --nota: double '%%' no final obrigatório:
+    AEBS_temperature = "temperatura média %s. Mínimo: %s. Máximo: %s. Umidade: %s%%...",
+    AEBS_wind_0 = "%s vento do %s. Máximo de %s esperado. %s",
+    AEBS_wind_1 = "Suave",
+    AEBS_wind_2 = "Moderado",
+    AEBS_wind_3 = "Forte",
+    AEBS_wind_4 = "Força da tempestade",
+    AEBS_clouds_0 = "Céu limpo.",
+    AEBS_clouds_1 = "Algumas nuvens.",
+    AEBS_clouds_2 = "Coberto de nuvens pesadas.",
+    AEBS_clouds_3 = "Períodos nublados periódicos.",
+    AEBS_clouds_4 = "Períodos de nuvens pesadas.",
+    AEBS_fog_0 = "Nevoeiro leve.",
+    AEBS_fog_1 = "Névoa espessa.",
+    AEBS_fog_2 = "Névoa muito espessa.",
+    AEBS_weather_0_a = "AVISO: Período de tempo previsto para começar em %s... ",
+    AEBS_weather_0_b = "A previsão do tempo continua ao longo do dia... ",
+    AEBS_weather_0_c = "Período meteorológico previsto para terminar em %s... ",
+    AEBS_weather_predicted = "espectativa ",
+    AEBS_weather_and_a = " e um ",
+    AEBS_weather_light_moderate = "espectativa de chuva fraca a moderada...",
+    AEBS_weather_heavy_rain = "pé d'água ou chuva forte",
+    AEBS_weather_storm = "tempestade",
+    AEBS_weather_tropical = "tempestade tropical",
+    AEBS_weather_blizzard = "nevasca",
+    AEBS_weather_snowfall = " com chance de cair de neve.",
+    AEBS_weather_storm_C = "Tempestade",
+    AEBS_weather_tropical_C = "Tempestade tropical",
+    AEBS_weather_blizzard_C = "Nevasca",
+    AEBS_weather_warning = "Alerta de tempo severo. %s iminente. ETA %s dias...",
```

Comment:
**Alerta de Tempo Severo.** %s **Iminente**. **Tempo Estimado de Chegada:** %s dias...

ETA = Estimated Time of Arrival